package.json

```diff
@@ -19,8 +19,18 @@
 		"@types/hapi": "^18.0.3",
 		"@types/hapi__joi": "^16.0.12",
 		"eslint": "^6.8.0",
+		"husky": "^4.2.5",
 		"jest": "^25.2.7",
+		"lint-staged": "^10.1.3",
 		"nodemon": "^2.0.3",
 		"prettier": "^2.0.4"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
+		}
+	},
+	"lint-staged": {
+		"*.js": "eslint --cache --fix"
```

Comment:
Should we run our format and lint commands instead of eslint directly?

Reply:
Yep, of couse you should add all you need there, I added that command with one pattern because I don't know what lint command you need at this moment.

Reply:
He means these two https://github.com/streamdevs/webhook/blob/master/package.json#L8-L9 @hugolesta